lib/events/dynamoevents/dynamoevents.go

```diff
@@ -134,6 +144,15 @@ func (cfg *Config) SetFromURL(in *url.URL) error {
 		cfg.Endpoint = endpoint
 	}
 
+	const boolErrorTemplate = "failed to parse URI %q flag %q - %q, supported values are 'true' or 'false'"
```

Comment:
I realize that this is old and you didn't come up with it, but technically this error message is incorrect. `strconv.ParseBool` handles more than `true` or `false`:

```
➜ go doc strconv.ParseBool
package strconv // import "strconv"

func ParseBool(str string) (bool, error)
    ParseBool returns the boolean value represented by the string. It accepts 1,
    t, T, TRUE, true, True, 0, f, F, FALSE, false, False. Any other value
    returns an error
```